core-primitives/stf-executor/src/error.rs

```diff
@@ -28,7 +28,7 @@ pub type Result<T> = core::result::Result<T, Error>;
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
 	#[error("Trusted operation has invalid signature")]
```

Comment:
The corresponding `Display` implementation should also be changed, i.e., the text in the error macro.